api/v1alpha1/tracing_types.go

```diff
@@ -114,3 +116,16 @@ type RequestHeaderCustomTag struct {
 	// +optional
 	DefaultValue *string `json:"defaultValue,omitempty"`
 }
+
+// ZipkinTracingProvider defines the Zipkin tracing provider configuration.
+type ZipkinTracingProvider struct {
+	// TraceID128Bit determines whether a 128bit trace id will be used
+	// when creating a new trace instance. If set to false, a 64bit trace
+	// id will be used.
+	// +optional
+	TraceID128Bit *bool `json:"traceId128Bit,omitempty"`
```

Comment:
I understand this is creating a 1:1 mapping with envoy https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/trace/v3/zipkin.proto#extension-envoy-tracers-zipkin but this name is confusing
prefer if this was `TraceIDSize/TraceIDLength *TraceIDType` which could default to `64`

Reply:
Would `EnableTraceID128Bit` or `Enable128BitTraceID` be a better name?

Reply:
yeah I'm a +1 for `Enable128BitTraceID` wdyt @envoyproxy/gateway-maintainers @basvanbeek

Reply:
`Enable128BitTraceID` has my preference too.

Reply:
Thank you for input. Field name updated